A5_2practice_2.java

```diff
@@ -0,0 +1,55 @@
+import java.util.Scanner;
+public class A5_2practice_2 {
+
+	public static void main(String[] args) {
+		
+		boolean isPrime;
+		boolean isPerfect;
+	
+		Scanner scan= new Scanner(System.in);
+		System.out.println("Enter any number:");
+		//capture the input in an integer
+		int n=scan.nextInt();

```

Comment:
you need a loop that "repeatedly display a menu allowing the user to choose whether to display the prime numbers 1-1000, display the perfect numbers 1-10,000, or quit"

Reply:
you can do a prompt such as:
"type 1 to display if the number is prime.
type 2 to display if the number is perfect
type 0 to quit"
then if the user enter 1 or 2 then prompt:
"type your number"
and call the right method (isPrime or isPerfect).